tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,6 +127,68 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_revert_checks(self):
+        self.git_commit('Test commit')
+        commit_hash, _ = run_shell_command('git rev-parse HEAD')
+        self.git_commit('Revert "Test commit"\n\n'
+                        'Abcdef blablabla commit ' + str(commit_hash) + '\n\n'
+                        'Reason for the revert commit is explained here.\n')
+        self.assertEqual(self.run_uut(),
+                         ['Invalid revert commit.'])
+        self.assert_no_msgs()
+
+        self.git_commit('Test commit1\n\n'
+                        'This is body of test commit1\n')
+        commit_hash, _ = run_shell_command('git rev-parse HEAD')
+
+        self.git_commit('Revert "Another Test commit2"\n\n'
```

Comment:
This test method contains many testcases of revert features. I think it will be better if we can split this into several test method like `test_revert_check_success`, `test_revert_check_commit_no_reason`, `test_revert_check_invalid_sha_commit`, etc